ndcube/tests/test_ndcubesequence.py

```diff
@@ -138,13 +138,13 @@ def test_slice_first_index_sequence(test_input, expected):
 
 
 @pytest.mark.parametrize("test_input,expected", [
-    (seq.index_as_cube[0:5].dimensions, (3*u.pix, [2., 2., 1.]*u.pix, 3*u.pix, 4*u.pix)),
-    (seq.index_as_cube[1:3].dimensions, (2*u.pix, 1*u.pix, 3*u.pix, 4*u.pix)),
```

Comment:
Why remove this test case?

Reply:
Seems like the old cases has dimensions not valid after some changes are made, hence the proposed changes to make things work.

Reply:
Will follow it up with the cases where first slicing index starts at `1`

Reply:
What's the status on this issue?

Reply:
Let me check things again...